### PR TITLE
Card

### PR DESCRIPTION
**Pull request by SilviaBarcellini**
Add basic card component
- create basic element card
- create card component
- add style and hoover effect

### CODE REVIEW

File: components/footer/favs/favsBtn.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  component: favsBtn,
+} as Meta;
+
+const image = {
+  src: favsBtn,
+  alt: "add-to-favs",
+};
+
+export const favsImg = () => <img src={image.src} alt={image.alt} />;
+ */
```
Comment (lmachens) [inline components/footer/favs/favsBtn.stories.tsx:16]: If you don't need this code, better remove it from this pull request :) 
Same with the HomeBtn

File: components/greetings/greetings.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+if (currentTime < 12) {
+  greetings = "Good Morning";
+} else if (currentTime < 18) {
+  greetings = "Good Afternoon";
+} else if (currentTime < 22) {
+  greetings = "Good Evening";
+} else {
+  greetings = "Good Night";
+}
```
Comment (lmachens) [inline components/greetings/greetings.tsx:17]: Good job!
I would create a function from this (maybe `createGreetings()`?) and call it in the `Greetings` component.

File: components/footer/home/homeBtn.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import styles from "./homeBtn.module.css";
+
+/* import homeImg from "./homeBtn.stories"; */
+
+export type HomeBtnProps = {
+  src: string;
+};
+
+function HomeBtn({ ...props }: HomeBtnProps) {
```
Comment (lmachens) [inline components/footer/home/homeBtn.tsx:9]: Actually, this is not a button, but it's called `HomeBtn`.

**Review by lmachens — COMMENTED** [review]
good job, I would love to see a deployment

File: components/greetings/greetings.tsx (hunk 2 above)
Comment (SilviaBarcellini) [reply components/greetings/greetings.tsx:17]: Thank you! I will open a Pull Request for it

File: components/footer/home/homeBtn.tsx (hunk 3 above)
Comment (SilviaBarcellini) [reply components/footer/home/homeBtn.tsx:9]: True, it is a link. It is confusing, I am going to rename it. Thanks

**Pull request merged**